### DISCUSSION TITLE
Next version - any news yet ?

### DISCUSSION THREAD

**Discussion by Manfredo1952** [OrcaSlicer/OrcaSlicer#223] [General]
Hi there

Just a small question - any news regarding a new version of your nice piece of artwork ?

regards, Manfred

**Comment by SoftFever** [comment]
Very soon ;)

**Comment by Manfredo1952** [comment]
Aha ....  
set(SLIC3R_VERSION "01.04.02.13")
set(SoftFever_VERSION "1.4.3")    

👍

**Comment by Manfredo1952** [comment]
Thank You !
